assignment_5.py

```diff
@@ -0,0 +1,204 @@
+#travelling
+
+class Travelling:
+     
+    def __init__(self,destination,route,SaftyGears,accomodation,budget):
+        self.destination = destination
+        self.route = route
+        self.SaftyGears = SaftyGears
+        self.accomodation = accomodation
+        self.budget = budget
+
+    def PlanTrip(self):
+        print("Planning bike trip to:",self.destination)
+        print("1. route:",self.route)
+        print("2. accomodation:",self.accomodation)
+        print("3. Safty Precautions:",self.SaftyGears)
+        print("4. Total budget estimation:",self.budget)
+
+destination = "Ladakh"
+route = "By Road from Tamil Nadu - vellore to Ladakh "
+SaftyGears = "Helmet, Riding Jacket, Boots, Glows, Thermal Liner "
+accomodation = "campimg along the way"
+budget = "The estimated budget for the trip is 4 lakhs"
+
+TripPlan = Travelling(destination, route, SaftyGears, accomodation, budget)
+TripPlan.PlanTrip()
+
+
+
+class Destinations():
+    def __init__(self,staring_point):
+        self.stating_point = staring_point
+        self.destinations = []
+
+    def AddDestination(self,destination):
+        self.destinations.append(destination)
+
+    def StartTrip(self):
+        print("Staring bike trip from", self.stating_point)
+
+        for i in range(len(self.destinations) - 1):
+            print("Biking from", self.destinations[i], "to" ,self.destinations[i+1])
+        print("The final destination gonna be ", self.destinations[-1])
+
+
+trip = Destinations("Vellore")
+trip.AddDestination("Kerala")
+trip.AddDestination("Karnataka")
+trip.AddDestination("Andhra Pradesh")
+trip.AddDestination("Odisha")
+trip.AddDestination("West Bengal")
+trip.AddDestination("Uttar Pradesh")
+trip.AddDestination("Rajasthan")
+trip.AddDestination("Delhi")
+trip.AddDestination("Jammu Kashmir")
+trip.AddDestination("Ladakh")
+
+trip.StartTrip()
+
+
+
+from bs4 import BeautifulSoup
+
+soup = BeautifulSoup(flipcart.html, 'html.parser')
+
+def finf_name():
+    print(soup.text, "soup text")
+    product_name = soup.find(class_="span.B_NuCI")
```

Comment:
try some other website. this is pretty simple. checkout cricbuzz or some other websites.